_sub/database/postgres/main.tf

```diff
@@ -21,9 +21,9 @@ resource "aws_security_group" "pgsg" {
 
 #Enable SSL on the database by default
 resource "aws_db_parameter_group" "dbparams" {
-  name        = "${var.application}-postgres10-force-ssl-${var.environment}"
-  description = "Force SSL encryption for postgres10"
-  family      = "postgres10"
+  name        = "${var.application}-${var.db_parameter_group_family}-force-ssl-${var.environment}"
+  description = "Force SSL encryption for ${var.db_parameter_group_family}"
+  family      = var.db_parameter_group_family
```

Comment:
No need to introduce a new variable that is already a combination of var.engine and var.engine_version. Instead create a locals and combine the two variables into a engine_familiy variable.

Reply:
Also you forgot about the naming of the security group above in line 2.

Reply:
Engine version can also include a minor version, so by doing it this way we restrict ourselves from using those, ie. engine_version 13 will be ok, but not 13.1 if we ever had to run that specific version (although the default is to allow minor version upgrades, so 13 automatically means latest minor 13 version). Are we ok with that? (I am)

Another thing, how certain are we that the family name will always be `${engine}${engine_version}`? This way of doing things will break if AWS brings out family postgres13.5 or something like that, which they are doing on mysql. Looks like the postgresql family names have been like this since v11, so maybe we can assume it wil stay this way?

If we are ok with those two limitations, I will go ahead and refactor the PR with the approach you outlined.

Reply:
About the security group, it does not have anything to do with the engine version in the first place, so I would actually prefer either just renaming it from `postgres10` to `postgres`, or just leaving it as it is.

Reply:
Resolved after call